posthog/api/personal_api_key.py

```diff
@@ -39,24 +97,13 @@ def create(self, validated_data: dict, **kwargs) -> PersonalAPIKey:
         return personal_api_key
 
 
-class PersonalAPIKeyViewSet(
-    mixins.ListModelMixin,
-    mixins.RetrieveModelMixin,
-    mixins.CreateModelMixin,
-    mixins.DestroyModelMixin,
-    viewsets.GenericViewSet,
-):
+class PersonalAPIKeyViewSet(viewsets.ModelViewSet):
```

Comment:
One last security aspect: let's remove `PersonalAPIKeyAuthentication` altogether from this endpoint. It should never be possible to create or edit a personal API key using another personal API key, because this would allow bypassing the whole scoping mechanism

Reply:
When testing I just found that it's possible to create personal API keys with _any_ personal API key, no matter the allowed scope

Reply:
Added a test to cover this and removed the default auth to make sure this sort of thing couldn't happen in the future